pages/linux/pactree.md

```diff
@@ -0,0 +1,24 @@
+# pactree
+
+> Package dependency tree viewer for pacman.
+> More information: <https://man.archlinux.org/man/community/pacman-contrib/pactree.8.en>.
+
+- View the dependency tree of a package:
+
+`pactree {{package}}`
+
+- View what packages depend on this package:
+
+`pactree --reverse {{package}}`
+
+- List dependencies at the start of the line and only with unique entries:
```

Comment:
```suggestion
- List dependencies, one per line, and only with unique entries:
```

Reply:
I would argue against this because this is what it looks like normally:
```
firefox
├─gtk3
│ ├─atk
│ │ └─glib2
│ │   ├─pcre
```
and this is what it looks like with -u or -l
```
firefox
gtk3
atk
glib2
pcre
```

It's still one per line in both cases, but the formatting doesn't have any fancy graphics.

Reply:
We use `list` as verb in too many pages, the tree is not a common output from commands. Otherwise you can use `Print as list`

Reply:
```suggestion
- List dependencies without the tree graphics and only with unique entries:
```
How about something along the lines of this?

Reply:
How about
```suggestion
- Formlessly list dependencies, only showing unique entries:
```
or
```suggestion
- Formlessly list dependencies, skipping duplicates:
```

Reply:
I feel like "formlessly" describes the command poorly since it can be understood as "format" but also "the shape something". For me at least it feels really nondescript.

Reply:
Maybe `Dump dependencies`?

Reply:
You guys pick the wording on this one. I'm unable to decide.

Reply:
I'd go with this one if nobody objects:
```suggestion
- Dump dependencies one per line, skipping duplicates:
```

Reply:
@Managor could you please apply this?